q2_katharoseq/tests/test_method.py

```diff
@@ -8,8 +8,8 @@
 from qiime2 import NumericMetadataColumn
 
 from q2_katharoseq import read_count_threshold, estimating_biomass
-from q2_katharoseq import allosteric_sigmoid
-from q2_katharoseq import get_threshold
+from q2_katharoseq._methods import allosteric_sigmoid
```

Comment:
should these be added to `__init__.py` so they can be imported directly?

Reply:
they aren't intended to be used generally so import from private is fine